Makefile

```diff
@@ -7,12 +7,27 @@ $(error $(CONFIG_FILE) not found. See $(CONFIG_FILE).example.)
 endif
 include $(CONFIG_FILE)
 
+# Rectify input parameters
+ifeq ($(CPU_ONLY),1)
+  USE_CUDNN=0
+  USE_CNMEM=0
+endif
+
+ifeq ($(USE_CUDNN),1)
+# CNMEM is ON by default in CUDNN is ON
+  ifeq ($(USE_CNMEM),)
+    USE_CNMEM=1
+  endif
+endif
+
+PROJECT_DIR=$(PWD)
```

Comment:
I needed so set paths to absolute, let’s discuss this.

From: Luke Yeager [mailto:notifications@github.com]
Sent: Wednesday, October 21, 2015 5:38 PM
To: NVIDIA/caffe
Cc: Boris Fomitchev
Subject: Re: [caffe] CUDNN v4 upgrade. Optionally, CNMEM used for GPU pool allocation. (#42)


In Makefile<https://github.com/NVIDIA/caffe/pull/42#discussion_r42700544>:

> @@ -7,12 +7,27 @@ $(error $(CONFIG_FILE) not found. See $(CONFIG_FILE).example.)

>  endif

>  include $(CONFIG_FILE)

>

> +# Rectify input parameters

> +ifeq ($(CPU_ONLY),1)

> +  USE_CUDNN=0

> +  USE_CNMEM=0

> +endif

> +

> +ifeq ($(USE_CUDNN),1)

> +# CNMEM is ON by default in CUDNN is ON

> +  ifeq ($(USE_CNMEM),)

> +    USE_CNMEM=1

> +  endif

> +endif

> +

> +PROJECT_DIR=$(PWD)

@borisfom<https://github.com/borisfom> why did you add this PROJECT_DIR variable? It broke my debian packaging. Is it really necessary?

—
Reply to this email directly or view it on GitHub<https://github.com/NVIDIA/caffe/pull/42/files#r42700544>.

-----------------------------------------------------------------------------------
This email message is for the sole use of the intended recipient(s) and may contain
confidential information.  Any unauthorized review, use, disclosure or distribution
is prohibited.  If you are not the intended recipient, please contact the sender by
reply email and destroy all copies of the original message.
-----------------------------------------------------------------------------------